sections/variant-list.liquid

```diff
@@ -83,6 +88,13 @@
             </tbody>
           </table>
         </div>
+        <noscript>
+          {%- if product.has_only_default_variant or product.variants.size == 1 -%}
+            <button type="submit" class="button button--secondary right" formnovalidate form="VariantList">
```

Comment:
I dont think we should be disabling validation. You can keep it and do what we do in cart: https://screenshot.click/14-49-vko24-fjxlb.png

Let's show the trash icon even when JS is off

Reply:
I think the `formnovalidate` is to prevent variants with quantity = 0 to block form submission. Not sure if there are any other side-effects to this. I would prefer removing the `min` attribute on the quantity input for no-js and let the backend handle the minimum quantity validation.

Reply:
Sofia, is this comment was before we discussed validation? 

What's is the point to keeping trash icons if it's not working without js?